### PR TITLE
Changed general style

### PR DESCRIPTION
**Pull request by Fabrizz**
### Before:
image

### After:
image

### CODE REVIEW

**Comment by KristjanESPERANTO** [comment]
That looks really good! 👍

Is it possible without Google fonts? My idea was to use the same fonts as the standard MagicMirror fonts in order to have a certain consistency.

**Comment by Fabrizz** [comment]
Yeah, there are two options, serving them in a `/fonts` folder or just using the browser default (not MM² roboto).

I always download the fonts using https://gwfh.mranftl.com/fonts and serve them, Github caching should just work

**Comment by Fabrizz** [comment]
done

**Comment by KristjanESPERANTO** [comment]
Nice! :+1: 

Many tags are an issue on Firefox:

Screenshot from 2024-02-13 21-04-54

File: docs/styles.css [hunk 1]
```diff
@@ -91,200 +133,193 @@ a {
...
-[data-tag="calendar"]::before {
-  content: "📅";
-}
-
-[data-tag="soccer"]::before {
-  content: "⚽";
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 100%;
+  max-height: 200px;
```
Comment (KristjanESPERANTO) [inline docs/styles.css:270]: I would prefer _slightly_ larger images. What do you think about increasing it to 300px?

**Comment by KristjanESPERANTO** [comment]
After solving the tag issue I think it's ready to merge :smiley:

**Comment by Fabrizz** [comment]
> done, changes: https://fabrizz.github.io/MagicMirror-3rd-Party-Modules/

Changed the tags and img size, you probably need to clear the cache to see it `CTRL + F5`

File: docs/script.js [hunk 2]
```diff
@@ -220,18 +275,32 @@ moduleCardContainer.addEventListener("click", (event) => {
...
+  const root = document.querySelector(":root");
   filteredModuleList = allModules;
   searchInput.value = "";
   showOutdated.checked = true;
   removeSelectedMarkingFromTagsAndCards();
   sortDropdown.value = "default";
   sortData(sortDropdown.value);
+  root.style.setProperty("--color-accent-header", "var(--color-background)");
+
+  /* --color-accent-light: #ebf8ff; --color-accent-dark: #033454; */
```
Comment (KristjanESPERANTO) [inline docs/script.js:287]: Do we still need this deactivated part?

File: docs/script.js [hunk 3]
```diff
@@ -8,51 +9,94 @@ const tagButtonContainer = document.getElementById("tag-buttons");
 const sortDropdown = document.getElementById("sort-dropdown");
 const showOutdated = document.getElementById("show-outdated");
 
+/* [<name>, <icon>, <background>, <foreground>] */
+/* Add aliases to modules so similar can be found */
```
Comment (KristjanESPERANTO) [inline docs/script.js:13]: I don't get what you mean by aliases.

**Review by KristjanESPERANTO — APPROVED** [review]

File: docs/script.js (hunk 2 above)
Comment (Fabrizz) [reply docs/script.js:287]: No, the comment is just a reference

File: docs/script.js (hunk 3 above)
Comment (Fabrizz) [reply docs/script.js:13]: I did not look at the sorter code, so I dont know if its don by the module tags on github or by the section that they are in the3rd party module markdown  on mm2 repo. Some modules have tags that are similar and do not get sorted, the alias there would mean to rewritten everything of the sort part

Comment (KristjanESPERANTO) [reply docs/script.js:13]: The tags of the modules come from the keywords in the `package.json` of the respective repository. The tag buttons on the page is be something like the replacement for the categories of the module list in the wiki.

**Comment by KristjanESPERANTO** [comment]
Thanks for the lazy loading!

I really like your changes! Thanks a lot! 😀

I hereby merge 🚀

File: docs/script.js (hunk 2 above)
Comment (KristjanESPERANTO) [reply docs/script.js:287]: Okay, I just removed it :-)

**Pull request merged**